src/utils.ts

```diff
@@ -104,8 +104,17 @@ class Utils {
 		const platformConfig = this.getPlatformConfig(platformId);
 		const options: QueueSettings = platformConfig.queue.defaults;
 		if (!options.redis) { options.redis = {}; }
-		options.redis.host = process.env.REDIS_HOST || '127.0.0.1';
-		options.redis.port = parseInt(process.env.REDIS_PORT || '6379', undefined);
+		if (!options.redis.url) {
```

Comment:
Okay, I think that way was not my goal in the Ticket. After reading additional information http://www.iana.org/assignments/uri-schemes/prov/redis i thik it shoud be possible use an DNS name for the host enviorment variable.

Reply:
Sorry but I think that I am missing something. The final solution should be like storing DNS name in the environment variable which would be configurable in Dockerfile, which then would be assigned to the redis.url property instead of the host and port? If yes I would like to know from where can I obtain this DNS name.

Reply:
The environment variable will be set on the deployment to the target system, not in the dockerfile. For example: 
docker run -e REDIS_HOST=redis.test.dns.name -e REDIS_PORT=1234 schulcloud/node-notification-service:latest 

Maybe it shoud be possibel to set the DNS Name as REDIS_HOST. After i search for how the connection was triggert and how the connection function is uesd. I think my first opinion in the ticket to use the URL options was wrong.

Reply:
Ok, so I understand I should leave it as it as as it will be configured during the deployemnt stage.